scalableminds/webknossos
# Add Summary Row to Time Tracking

### URL of deployed dev instance (used for testing):
- https://sumtrackedtime.webknossos.xyz/

### Steps to test:
- Have some tracked times for different annotations/tasks/projects/users
- Go to time tracking overview (as admin) and check the summary row
- Make sure the sums stay up-to-date while you play around with the filters
- also filter sth. without any time tracking entries

### Issues:
- fixes #8088 

------
(Please delete unneeded items, merge only when none are left open)
